vendor/knative.dev/pkg/controller/stats_reporter.go

```diff
@@ -199,7 +199,7 @@ func (r *reporter) ReportReconcile(duration time.Duration, success string, key t
 		return err
 	}
 
-	// TODO skonto: fix latency histogram
-	metrics.RecordBatch(ctx, reconcileCountStat.M(1))
+	metrics.RecordBatch(ctx, reconcileCountStat.M(1),
+		reconcileLatencyStat.M(duration.Milliseconds()))
```

Comment:
Some missing patches to vendor

Reply:
MAKE RELEASE=1.11 is not giving me any diff